packages/design-picker/src/utils/designs.ts

```diff
@@ -77,3 +77,16 @@ export const getThemeIdFromDesign = ( design: Design ) => {
 	}
 	return null;
 };
+
+export const shuffleDesigns = ( designs: Design[], seed: number ) => {
+	const designsToShuffle = [ ...designs ];
+	for ( let i = designsToShuffle.length - 1; i > 0; i-- ) {
+		const j = ( i + seed + designsToShuffle.length ) % i;
```

Comment:
I think it would be better if the modulo is $i + 1$, to add the chance that the element is not swapped at all:

```suggestion
		const j = ( i + seed + designsToShuffle.length ) % (i + 1);
```

Reply:
Good point 👍